ecommerce-ui/src/App.js

```diff
@@ -0,0 +1,52 @@
+import classes from "./App.module.css"
+import { useState } from "react"
+import bnbData from "./data/bnbs.json"
+import VacationRental from "./components/VacationRental"
+import ShoppingCart from "./components/ShoppingCart"
+import Form from "./components/Form"
+import Button from "./ui/Button"
+
+function App() {
+  let [cart, setCart] = useState([])
+  let [bnbs, setBnbs] = useState(bnbData)
+  let [isShoppingCartDisplayed, setIsShoppingCartDisplayed] = useState(false)
+  function handleUserInput(userData) {
+    setBnbs(prevState => [...prevState, userData])
+  }
+  function handleAddToCart(bnbId) {
+    let itemToAdd = bnbs.find(item => item.id === bnbId)
+    let alreadyExist = cart.some(item => item.id === itemToAdd.id)
```

Comment:
nice check. good UI

There are a lot of let's on this component, most (if not all) could be used as a const. Always strive to use const over let if you can.